lib/mobility/plugins/original_column.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Mobility
+  module Plugins
+    module OriginalColumn
+      extend Plugin
+
+      default false
```

Comment:
The default is `false` which means that it needs to be explicitly passed `original_column: true` on every call to `translates`. I think this makes sense since this plugin incurs a perf hit on every read/write, and probaly people only want to use it selectively on some columns/models.

Reply:
Oh! This may be the right spot to get rid of that `I18n` dependency.
```
class Post
  extend Mobility
  translates :name, type: :string, original_column_fallbacks: [:en, :en-US]
end
```

Reply:
Yes indeed! I'll try that.